src/components/app/details/cdDetails/CDDetails.tsx

```diff
@@ -167,14 +172,16 @@ export default function CDDetails() {
                             />
                         </Route>
                     ) : !envId ? (
-                        <EmptyView
-                            title="No environment selected"
-                            subTitle="Please select an environment to start seeing CD deployments."
+                        <GenericEmptyState
+                            image={AppNotDeployed}
```

Comment:
We can remove image from here as this is the default image

Reply:
Done